### PR TITLE
Sorbus port rebase

### PR DESCRIPTION
**Pull request by BensonRSI**
port to sorbus system : https://xayax.net/sorbus/ 
working with DMA on a flash-filesystem, which gives real improvement in access and speed .
Also included a x-modem receiver which works on AUX devices. On Sorbus this  is the serial-interface, without character translation ( CR/LF off) , so you can push binaries over your terminal to the filesystem.

### CODE REVIEW

File: sorbus.zip
Comment (davidgiven) [inline sorbus.zip:1]: Is this file supposed to be here?

File: apps/drivers.inc [hunk 1]
```diff
@@ -13,6 +14,11 @@ TTY_CONST = 0
 TTY_CONIN = 1
 TTY_CONOUT = 2
 
+AUX_IN = 0
```
Comment (davidgiven) [inline apps/drivers.inc:17]: I actually want to rework this into a generic SERIAL driver, supporting things like changing baud rate, but I'll do that myself and make sure it's binary compatible with this.

File: sorbus.zip
Comment (BensonRSI) [reply sorbus.zip:1]: It's an artifact , from the build i removed it.

File: apps/drivers.inc (hunk 1 above)
Comment (BensonRSI) [reply apps/drivers.inc:17]: That's also a good idea, you can add a "transparent"- mode to the consoleIO. My initial thoughts are, that systems with an extra serial-interface could use "AUX" , but that doesn't really make sense :(

**Comment by davidgiven** [comment]
That's merged, thanks! However, I could use some installation and usage instructions to go into the README.

**Comment by BensonRSI** [comment]
Hi David,
i talked to SvOlli, he will extend the Readme and provide some
installation-instructions.

Cheers,
Olli


Am 20.12.23 um 21:30 schrieb David Given:

**Comment by davidgiven** [comment]
Done --- thanks!

**Pull request merged**